packages/ui/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,220 @@
+                >
+                    {Boolean(
+                        categories.length ||
+                            hasStandaloneAboutPage ||
+                            hasStandaloneContactsPage ||
+                            publicGalleriesCount,
+                    ) && (
+                        <div className="pt-6 md:pt-0 flex flex-col md:flex-row md:items-center gap-12 md:gap-4 px-6 md:px-0">
+                            {categories.length > 0 && (
+                                <CategoriesDropdown categories={categories} locale={locale} />
+                            )}
+                            {publicGalleriesCount > 0 && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
+                                    href="/media"
+                                    locale={locale ?? false}
+                                >
+                                    {/* TODO: Use translations */}
+                                    Media
+                                </Link>
+                            )}
+                            {hasStandaloneAboutPage && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
+                                    href="/about"
+                                    locale={locale ?? false}
+                                >
+                                    {/* TODO: Use translations */}
+                                    About
+                                </Link>
+                            )}
+                            {hasStandaloneContactsPage && (
+                                <Link
+                                    className={twMerge(
+                                        'label-large text-gray-600 hover:text-gray-800 shrink-0',
+                                        !isSm && `text-lg font-bold`,
+                                    )}
+                                    href="/contacts"
+                                    locale={locale}
+                                >
+                                    {/* TODO: Use translations */}
+                                    Contacts
+                                </Link>
+                            )}
+                        </div>
+                    )}
+
+                    <div className="flex flex-col md:flex-row md:items-center gap-12 md:gap-4">
+                        {Boolean(onSearch) && (
+                            <a className="hidden md:flex" href="#" onClick={handleSearch}>
```

Comment:
Why is this a link and not a button? You're using the button in the mobile markup below 🤔